code/datums/wires/wires.dm

```diff
@@ -121,15 +121,23 @@ var/list/wireColours = list("red", "blue", "green", "black", "orange", "brown",
 			var/obj/item/I = L.get_active_hand()
 			holder.add_hiddenprint(L)
 			if(href_list["cut"]) // Toggles the cut/mend status
-				if(iswirecutter(I))
+				var/switchtool_deployed
```

Comment:
Setting `I` equal to `S.deployed` would actually be even better than the way I said to do it, I simply didn't think of it at the time.
It would allow you to get rid of the `var/switchtool_deployed` definition as well as the extra check in the `if()` statement below.